tests/unit_tests/commandline/cli_top_level_tester.py

```diff
@@ -23,7 +24,7 @@ class BaseCliTest(abc.ABC):
     _get_interface_args = ["geoips", "get", "interface"]
```

Comment:
I think I'd like to move these onto each class. These are confusing where they are, though I do see how they simplify things a bit.

Reply:
I moved these on each class in #644